[torchgen] Fix selective build error on custom namespace

larryliu0820 opened the pull request:
Summary: Currently `SelectiveBuilder` is hardcoding namespace `aten` for operators. This is not working anymore since operators started to have custom namespaces. This fixes it.

Test Plan: Rely on newly added unit test

Differential Revision: D38565527

JacobSzwejbka reviewed (approved):
fix lints but otherwise lgtm

larryliu0820 committed:
[torchgen] Fix selective build error on custom namespace (#83141)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/83141

Currently `SelectiveBuilder` is hardcoding namespace `aten` for operators. This is not working anymore since operators started to have custom namespaces. This fixes it.

Test Plan: Rely on newly added unit test

Reviewed By: JacobSzwejbka, iseeyuan

Differential Revision: D38565527

fbshipit-source-id: b665100e2130eed36b46866fb187cb63105546b9
